NineChronicles.Headless.Tests/GraphTypes/ActionQueryTest.cs

```diff
@@ -1340,5 +1342,81 @@ public async Task AuraSummon()
+            var claimDataList = new List<(Address, List<FungibleAssetValue>)>();
+            var queryBuilder = new StringBuilder().Append("{claimItems(claimData: [");
+            for (var i = 0; i < claimDataCount; i++)
+            {
+                var avatarAddress = new PrivateKey().ToAddress();
+                var currencyCount = random.Next(1, tickerList.Count + 1);
+                var tickerCandidates = tickerList.OrderBy(i => random.Next()).Take(currencyCount);
+                queryBuilder.Append($@"{{
+                    avatarAddress: ""{avatarAddress}"",
+                    fungibleAssetValues:[
+                ");
+                var favList = tickerCandidates.Select(
+                    ticker => new FungibleAssetValue(
+                        Currency.Uncapped(
+                            ticker: ticker,
+                            decimalPlaces: 0,
+                            minters: new AddressSet(new List<Address>())
+                        ),
+                        random.Next(1, 100), 0
+                    )
+                ).ToList();
+                foreach (var fav in favList)
+                {
+                    queryBuilder.Append($@"{{
+                        ticker: ""{fav.Currency.Ticker}"",
+                        decimalPlaces: 0,
+                        minters: [],
+                        quantity: {fav.MajorUnit}
+                    }}");
+                    if (fav != favList[^1])
+                    {
+                        queryBuilder.Append(",");
+                    }
+                }
+
+                claimDataList.Add((avatarAddress, favList));
+
+                queryBuilder.Append("]}");
+                if (i < claimDataCount - 1)
+                {
+                    queryBuilder.Append(",");
+                }
+            }
+
+            queryBuilder.Append("])}");
+
+            var queryResult =
+                await ExecuteQueryAsync<ActionQuery>(queryBuilder.ToString(), standaloneContext: _standaloneContext);
+            var data = (Dictionary<string, object>)((ExecutionNode)queryResult.Data!).ToValue()!;
+            var plainValue = _codec.Decode(ByteUtil.ParseHex((string)data["claimItems"]));
+            Assert.IsType<Dictionary>(plainValue);
+            var actionBase = DeserializeNCAction(plainValue);
+            var action = Assert.IsType<ClaimItems>(actionBase);
+
+            for (var i = 0; i < claimDataList.Count; i++)
+            {
+                var (expectedAddr, expectedFavList) = claimDataList[i];
+                var (actualAddr, actualFavList) = action.ClaimData[i];
+                Assert.Equal(expectedAddr, actualAddr);
+                Assert.Equal(expectedFavList.Count, actualFavList.Count);
+                for (var j = 0; j < expectedFavList.Count; j++)
+                {
+                    // Assert.Equal(expectedFavList[i], actualFavList[i]);
```

Comment:
Why is this line disabled? (by comment)

Reply:
That is not working because Currency hash of two FAVs are different.
I found the reason was different minters, I provided `[]` but currency from action has `null`
I tried to give null as a minters parameter but GQL does not allow this. So I gave up to fix this :fearful:

Reply:
Yes, it is GraphType's bug... 🐛  I think I can fix it easily but I'm scared if it makes a huge breaking-change. At this time, I understood why the line is disabled.

Reply:
I'll add additional comment on top of the this line why this is disabled and how to fix this :+1: